src/main/java/core/basesyntax/service/impl/TransactionServiceImpl.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.TransactionService;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TransactionServiceImpl implements TransactionService {
+    private static final String LINE_SEPARATOR = ",";
+    private static final int FRUIT_NAME = 1;
+    private static final int FRUIT_QTY = 2;
+    private static final int TRANSACTION_CODE = 0;
```

Comment:
Consider renaming the constant 'TRANSACTION_CODE' to 'TRANSACTION_CODE_INDEX' for better clarity.